extensions/ql-vscode/.eslintrc.js

```diff
@@ -31,8 +38,32 @@ module.exports = {
     "@typescript-eslint/no-non-null-assertion": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-floating-promises": [ "error", { ignoreVoid: true } ],
+    "@typescript-eslint/no-invalid-this": "off",
+    "@typescript-eslint/no-shadow": "off",
     "prefer-const": ["warn", { destructuring: "all" }],
     "@typescript-eslint/no-throw-literal": "error",
     "no-useless-escape": 0,
+    "camelcase": "off",
+    "eqeqeq": "off",
+    "escompat/no-regexp-lookbehind": "off",
+    "filenames/match-regex": "off",
+    "filenames/match-regexp": "off",
+    "func-style": "off",
+    "i18n-text/no-en": "off",
+    "import/named": "off",
+    "import/no-dynamic-require": "off",
+    "import/no-dynamic-required": "off",
+    "import/no-anonymous-default-export": "off",
+    "import/no-commonjs": "off",
+    "import/no-mutable-exports": "off",
+    "import/no-namespace": "off",
+    "import/no-unresolved": "off",
+    "import/no-webpack-loader-syntax": "off",
+    "no-invalid-this": "off",
+    "no-fallthrough": "off",
+    "no-console": "off",
+    "no-shadow": "off",
+    "github/array-foreach": "off",
+    "github/no-then": "off",
```

Comment:
The warnings already work, but there are 1000+ of them at the moment. Since we run eslint as a precommit hook, I think it would be quite disruptive to have all the warnings. Even if it's just narrowed down to the particular files you're changing, it's might still be a lot of noise. I'm wondering if there's a way to run eslint only on the lines you've changed.